docs/specification/draft/server/index.mdx

```diff
@@ -37,3 +43,17 @@ Explore these key primitives in more detail below:
   />
   <Card title="Tools" icon="wrench" href="/specification/draft/server/tools" />
 </CardGroup>
+
+## Profiles
+
+Profiles provide a way for servers to declare conformance to a set of behaviors and properties.
```

Comment:
You're talking about generic behaviors and properties, but the minimum profile focuses on authorization. Does it mean that the server needs to define different profiles for different things?

Reply:
Great point. I've adjusted the spec to say that a server can support multiple profiles simultaneously.